jplag/src/main/java/de/jplag/Submission.java

```diff
@@ -35,6 +35,11 @@
      */
     private final File submissionRootFile;
 
+    /**
+     * Whether the submission must be checked for plagiarism.
```

Comment:
Here we definitely should talk about old and new submissions. E.g. add: "old submissions, e.g. from previous years, are checked against but not with each other."